src/main/groovy/de/iteratec/osm/result/dao/EventResultQueryBuilder.groovy

```diff
@@ -110,6 +110,14 @@ class EventResultQueryBuilder {
         return withAssociatedDomainIdsIn(pages.collect { it.ident() }, 'page', project)
     }
 
+    EventResultQueryBuilder withoutPageIn(List<Long> pages) {
+        return withAssociatedDomainIdsNotIn(pages.collect { it.ident() }, 'page')
```

Comment:
`pages` seems to be a List of `Long`s, so probably the method signature is wrong and `List<Page>` was intended?